### ISSUE TITLE
Compile errors

### ISSUE THREAD

**Issue by fuzzthink** [clio-lang/clio#23]
src $ clio compile eg.clio out.js
```
/Users/johnleung/.nvm/versions/node/v10.14.1/lib/node_modules/clio-lang/index.js:30
  fs.writeFileSync(path, source);
  ^

ReferenceError: fs is not defined
```

eg.clio is just `'2' -> print`

**Comment by fuzzthink** [comment]
Same error after installing and using the following versions of node:

node v10.15.3 (npm v6.4.1)  -- LTS
node v11.11.0 (npm v6.7.0)

**Comment by pouya-eghbali** [comment]
Apparently I forgot to require `fs` there (this is probably because of code refactoring / lots of code rewrites there) will fix after I replace the old parser with the new one

**Comment by garritfra** [comment]
Will be fixed by #17
